api/envoy/extensions/http/stateful_session/header/v3/header.proto

```diff
@@ -76,11 +76,10 @@ message Mode {
   //
   // .. code-block:: none
   //
-  //   session-header: "MS4yLjMuNDo4MDtVVjp4eHh4eHgK" # base64("1.2.3.4:80;UV:xxxxxx")
+  //   session-header: "MS4yLjMuNDo4MDtVVjplSGg0ZUhoNENnPT0K" # base64(1.2.3.4:80;UV:base64(xxxxxx))
```

Comment:
I think this solve an issue if the `xxxxx` part contains a substring that may interfere with the Envoy filter processing.
I wonder if it would make sense to use the following pattern instead:
`base64(1.2.3.4:80);UV:base64(xxxxxx)`

The reasons are:
1) minor perf (no need to encode the ';UV:', and to the best of my knowledge the ';' and ':' will not appear in base64(xxxxxx).
2) easier debugging if the header is printed out (say with curl), as one can see that UV mode is used.

I think nesting should still work (if I'm not missing something), so for example if "xxxxx" = "abc;UV:xyz", it will still be encoded as <base64("1.2.3.4:80")>;UV:<base64("abs;UV:xyz")>.

Reply:
SGTM.